### PR TITLE
Add Dinner Menu

### PR DESCRIPTION
**Pull request by forfeen**
I'm recently add Dinner Menu

### CODE REVIEW

**Comment by KongtappV** [comment]
Dinner Menu did not link to `Menu.md`
try using: `[Dinner Menu](Menu.md#dinner-menu)`

**Comment by forfeen** [comment]
I already fixed it. Thanks !

File: README.md [hunk 1]
```diff
@@ -3,9 +3,13 @@
 **TODO**: add hyperlinks and format these one per line. Remove this TODO.
```
Comment (NarawishS) [inline README.md]: please remove TODO

Comment (forfeen) [reply README.md]: I already removed it. Thank you!

**Comment by forfeen** [comment]
I fixed all issues.

**Pull request merged**